Resources/Computing/GlobusComputingElement.py

```diff
@@ -0,0 +1,283 @@
+########################################################################
+# File :   GlobusComputingElement.py
+# Author : A.S.
+########################################################################
+
+""" Globus Computing Element
+
+   Allows direct submission to Globus Computing Elements with a SiteDirector Agent
+
+"""
+
+__RCSID__ = "$Id$"
+
+from DIRAC.Resources.Computing.ComputingElement          import ComputingElement
+from DIRAC.Core.Utilities.Grid                           import executeGridCommand
+from DIRAC                                               import S_OK, S_ERROR
+
+from DIRAC.ConfigurationSystem.Client.Helpers.Registry   import getGroupOption
+from DIRAC.FrameworkSystem.Client.ProxyManagerClient     import gProxyManager
+from DIRAC.WorkloadManagementSystem.DB.PilotAgentsDB     import PilotAgentsDB
+
+
+import os, tempfile
```

Comment:
The PEP8 specifications that we try to follow more and more recommend to do the import on different lines
https://www.python.org/dev/peps/pep-0008/#imports